zerver/data_import/slack.py

```diff
@@ -1316,11 +1316,32 @@ def convert_bot_info_to_slack_user(bot_info: dict[str, Any]) -> ZerverFieldsT:
         "real_name": bot_info["name"],
         "is_integration_bot": True,
         "profile": {
-            "image_72": bot_info["icons"]["image_72"],
             "bot_id": bot_info["id"],
             "first_name": bot_info["name"],
         },
     }
+    if "image_72" in bot_info["icons"]:
+        # Otherwise, gravatar will be used.
+        bot_user["profile"]["image_72"] = bot_info["icons"]["image_72"]
+
+    return bot_user
+
+
+def make_delete_bot_dummy(bot_id: str) -> ZerverFieldsT:
```

Comment:
```suggestion
def make_deleted_bot_dummy(bot_id: str) -> ZerverFieldsT:
```

Reply:
oh yeah, renamed to `make_deleted_placeholder`, that should be better?